livekit_rtc.proto

```diff
@@ -193,3 +198,19 @@ message SubscribedQualityUpdate {
   string track_sid = 1;
   repeated SubscribedQuality subscribed_qualities = 2;
 }
+
+message TrackPermission {
+  string participant_sid = 1;
+  bool all_tracks = 2;
+  repeated string track_sids = 3;
+}
+
+message UpdateSubscriptionPermissions {
+  bool all_participants = 1;
+  repeated TrackPermission track_permissions = 2;
+}
+
+message SubscriptionPermissionUpdate {
+  string track_sid = 1;
```

Comment:
should we include a participant_sid too? to make it easier to identify

Reply:
Good idea. Added in this commit - https://github.com/livekit/protocol/pull/59/commits/2819d3265d70c8d1ad194731da32b0a05bcd86d1